README.md

```diff
@@ -8,63 +8,73 @@
 
 ## Table of Contents
 
-- [Requirements](#requirements)
-- [Installation](#installation)
-  - [Platform Packages](#platform-packages)
+- [Installing](#installing)
+  - [Requirements](#requirements)
+  - [From sources](#from-sources)
   - [Godep](#godep)
+  - [Platform Packages](#platform-packages)
   - [Cross Compilation](#cross-compilation)
-- [Configuration](#configuration)
-- [Usage](#usage)
+  - [API keys](#api-keys)
+  - [Scripts](#scripts)
+- [Using](#using)
+  - [Hyphens](#hypens)
   - [Initializing](#initializing)
   - [De Initializing](#de-initializing)
-  - [End to End Encryption](#end-to-end-encryption)
   - [Traversal Depth](#traversal-depth)
+  - [Configuring General Settings](#configuring-general-settings)
+  - [Excluding And Including Objects](#excluding-and-including-objects)
+    - [Sample .driveignore with the exclude and include clauses combined](sample-.driveignore-with-the-exclude-and-include-clauses-combined)
   - [Pulling](#pulling)
+    - [Verifying Checksum] (#verifying-checksum)
```

Comment:
Verifying checksums

Reply:
I'm not GOD just a Messenger from God￼ When I have your God around me you will never win￼